.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,71 @@
+name: Behavior Bug
+description: Report issues with behaviour bugs.
+labels: [ "bug" ]
+body:
+  - type: textarea
+    attributes:
+      label: Observed behavior
+      description: What you actually saw.
+    validations:
+      required: true
+      
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: What you expected to see.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: You may submit a video or an ordered list of the steps you took to find the issue.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Plugin list (Optional)
+      description: |
+        All plugins running on your server.
+        Use the command `/plugins`.
+    validations:
+      required: true
```

Optional shouldn't be required.
```suggestion
      required: false
```